File: helfomat-core/src/main/java/de/helfenkannjeder/helfomat/core/template/OrganizationTemplateRepository.java
package de.helfenkannjeder.helfomat.core.template;

import de.helfenkannjeder.helfomat.core.organization.OrganizationType;

/**
 * @author Valentin Zickner
 */
public interface OrganizationTemplateRepository {

    OrganizationTemplate findByOrganizationType(OrganizationType organizationType);

}

File: helfomat-core/src/main/kotlin/de/helfenkannjeder/helfomat/core/template/OrganizationTemplateRepository.kt
package de.helfenkannjeder.helfomat.core.template

import de.helfenkannjeder.helfomat.core.organization.OrganizationType

/**
 * @author Valentin Zickner
 */
interface OrganizationTemplateRepository {
    fun findByOrganizationType(organizationType: OrganizationType): OrganizationTemplate?
}